### DISCUSSION TITLE
Get-PnPSite Get-SPOSite LastItemModifiedDate LastItemUserModifiedDate LastContentModifiedDate

### DISCUSSION THREAD

**Discussion by YuriySamorodov** [pnp/powershell#166] [Q&A] [answered]
Hi,

I need some kind of assistance to understand the difference between 
```
TypeName: Microsoft.Online.SharePoint.PowerShell.SPOSite

Name                    MemberType Definition
----                    ---------- ----------
LastContentModifiedDate Property   datetime LastContentModifiedDate {get;}
```

```
TypeName: Microsoft.SharePoint.Client.Web

Name                     MemberType Definition
----                     ---------- ----------
LastItemModifiedDate     Property   datetime LastItemModifiedDate {get;}
```

and 
```
TypeName: Microsoft.SharePoint.Client.Web

Name                     MemberType Definition
----                     ---------- ----------
LastItemUserModifiedDate Property   datetime LastItemUserModifiedDate {get;}
```

I see those properties return different values:
![image](https://user-images.githubusercontent.com/5260172/105709460-cb59fb80-5f26-11eb-8185-c7a13fcb5fd3.png)
![image](https://user-images.githubusercontent.com/5260172/105709513-de6ccb80-5f26-11eb-96ea-606b9bc084dd.png)

Your help is greatly appreciated!

**Comment by erwinvanhunen** [comment, marked as answer]
The first: the last modified date of any content in any of the webs that make up the site collection
The second: the last modified date of items in that specific web. (e.g. the first one is an aggregation of these for each web)
The last: the last modified date a -user- has been modifying an item in the web. The first two also report changes done by the system/server.

**Reply by YuriySamorodov** [reply]
Thanks @erwinvanhunen. That's extremely helpful!
